kibbeh/src/modules/dashboard/ProfileBlockController.tsx

```diff
@@ -33,18 +34,21 @@ export const ProfileBlockController: React.FC<ProfileBlockControllerProps> = ({}
   const badges: badge[] = [];
   if (conn.user.staff) {
     badges.push({
-      content: "ƉS",
+      content: <StaffBadge />,
       variant: "primary",
       color: "white",
       title: t("components.userBadges.dhStaff"),
+      naked: true,
     });
   }
+
   if (conn.user.contributions > 0) {
     badges.push({
-      content: "ƉC",
+      content: <ContributorBadge contributions={conn.user.contributions} />,
       variant: "primary",
       color: "white",
       title: t("components.userBadges.dhContributor"),
```

Comment:
although you should probably add the no. of contributions to the title, so ppl can see it on hover

Reply:
```suggestion
      title: `${t("components.userBadges.dhContributor")} (${conn.user.contributions} ${t("pages.admin.contributions")})`,
```